thinkcompany/think-ui-library
# Feature/tco 335 add button links

### Feature Status

### Description of Changes
Describe the reason the code is being changed, and your approach to making the changes.

Ths PR updates the buttons to match the design styles, removes the button size options in favor of responsive buttons, and adds background color to stories where the button would be otherwise not visible. It uses the `type-link` mixin for the link styles(thanks Ian!)

- Link to Jira ticket: https://thinkbrownstone.atlassian.net/browse/TCO-335
- Link to Abstract ticket
- Link to other documentation

#### Screenshots
Include any screenshots if they are relevant.

![Screen Shot 2020-08-07 at 1 41 10 PM](https://user-images.githubusercontent.com/11634849/89673520-8973c500-d8b4-11ea-85c4-de7b99d0442c.png)![Screen Shot 2020-08-07 at 1 41 15 PM](https://user-images.githubusercontent.com/11634849/89673505-7cef6c80-d8b4-11ea-8db3-44f7b167b4dd.png)![Screen Shot 2020-08-07 at 1 41 22 PM](https://user-images.githubusercontent.com/11634849/89673515-88429800-d8b4-11ea-8117-741d5ec3db29.png)![Screen Shot 2020-08-07 at 1 48 05 PM](https://user-images.githubusercontent.com/11634849/89673653-b88a3680-d8b4-11ea-9170-b5987ab8519c.png)![Screen Shot 2020-08-07 at 1 48 11 PM](https://user-images.githubusercontent.com/11634849/89673656-ba53fa00-d8b4-11ea-8930-eeac633e3550.png)

![Screen Shot 2020-08-07 at 1 48 41 PM](https://user-images.githubusercontent.com/11634849/89673659-bc1dbd80-d8b4-11ea-81dc-106a33911baa.png)



### How to Review
- pull down this branch
- `npm start`
- check out the code/stories for the buttons

### Merge Checklist:

### GIF Description
Share a fun gif that describes this PR & how it makes you feel.

![Welcome](https://media.giphy.com/media/xT5LMwGBmInO6eNhpm/giphy.gif)
